examples/connectivity/plot_connectivity_measures.py

```diff
@@ -0,0 +1,92 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
```

Comment:
>This idea of the example compares different measures. But, the end-result is shown as classification accuracy which is actually overtaken. Will it be good to show example only about measures ?

That's right, but I added this classification example to illustrate the advantage of using the robust dispersion measure over the other standard measures. May be I can add a comment on that?
>we can also reduce number of ADHD subjects which obviously can save time.

I can of course reduce the number of subjects (if  it makes sense to run a classification on 24 subjects for instance).